src/tools/pub/dependencies.md

```diff
@@ -429,33 +429,50 @@ depend on that package.
 These stale versions can have a negative impact on
 the stability, performance, and quality of apps.
 
-We recommend the following best practices for package dependencies:
-
-Use [caret syntax](#caret-syntax)
-: Specifying dependencies with version ranges is such as `^1.6.3`
-  is a good practice because it allows the pub tool to
-  select newer versions of the package when they become available.
-  Also, it places an upper limit on the allowed version,
-  based on an assumption that packages use [semantic versions][],
-  where any version of path versioned `1.x` is compatible,
-  but where a new version `2.x` would be a major upgrade
-  that isn't semantically compatible with `1.x` versions. 
-
-Depend on the latest stable package versions
-: Use [`pub upgrade`][] to update to the latest package versions
-  that your pubspec allows.
-  To identify dependencies in your app or package that
-  aren't on the latest stable versions,
-  use [`pub outdated`][].
-
-Test whenever you update package dependencies
-: If you run [`pub upgrade`][] without updating your pubspec,
-  the API should stay the same
-  and your code should run as before—but test to make sure.
-  If you modify the pubspec and update to a new major version,
-  then you might encounter breaking changes,
-  so you need to test even more thoroughly.
-
+We recommend the following best practices for package dependencies.
+
+### Use [caret syntax](#caret-syntax)
+
+Specifying dependencies with version ranges is such as `^1.6.3`
+is a good practice because it allows the pub tool to
+select newer versions of the package when they become available.
+Also, it places an upper limit on the allowed version,
+based on an assumption that packages use [semantic versions][],
+where any version of path versioned `1.x` is compatible,
+but where a new version `2.x` would be a major upgrade
+that isn't semantically compatible with `1.x` versions. 
+
+### Depend on the latest stable package versions
+
+Use [`dart pub upgrade`][] to update to the latest package versions
+that your pubspec allows.
+To identify dependencies in your app or package that
+aren't on the latest stable versions,
+use [`dart pub outdated`][].
+
+### Test whenever you update package dependencies
+
+If you run [`dart pub upgrade`][] without updating your pubspec,
+the API should stay the same
+and your code should run as before—but test to make sure.
+If you modify the pubspec and update to a new major version,
+then you might encounter breaking changes,
+so you need to test even more thoroughly.
+
+### Verify the integrity of downloaded packages
+
+When getting new dependencies, use the [`--enforce-lockfile`][]
+option to ensure the extracted package content matches
+the contents of the original archive.
+Without modifying the [lockfile][],
+this flag only resolves new dependencies if:
+* `pubspec.yaml` is satisfied
+* `pubspec.lock` is not missing
+* The packages' [content hashes][] match
```

Comment:
See [this comment](https://github.com/dart-lang/site-www/pull/4407/files#diff-ab1f08ea6448a1b6e767a02de133569ca302eea3fa3807931043da0e661da631R153).

Reply:
That comment also links back to this comment. I'm confused, is the issue with "resolve"? It's the normal definition of "resolve" applied to packages - you have a package already, and you're getting new info on it, so the system resolves the difference between new and existing to decide what's needed. This isn't the place to define package resolution

Reply:
Both places cover almost the same ground, but do so in different ways. This second copy block explains it _better_, so I'm unsure why they differ.